projects/wave-core-new/src/lib/operators/dialogs/histogram-operator/histogram-operator.component.html

```diff
@@ -45,6 +45,12 @@
         <div fxLayout="row" *ngIf="(form.controls.autoBuckets.valueChanges | async) === false">
             <mat-form-field fxFlex>
                 <input matInput type="number" placeholder="Number of Buckets" formControlName="numberOfBuckets" />
+                <mat-hint class="error" *ngIf="form.controls.numberOfBuckets.errors?.min || form.controls.numberOfBuckets.errors?.max">
+                    The number of buckets must be between 0 and 100.
```

Comment:
I guess we should define min and max in `histogram-operator.component.ts ` in order to ensure using the same value in the error message (template) and in the form definition.

Reply:
2d5bfc0ab5958370deb2a074152c5c6dc5ddef42
and 
ba9d0b2acf1262841474a4abd195afcd5ca59bdd